docs/reference/search/rrf.asciidoc

```diff
@@ -156,17 +162,26 @@ GET example-index/_search
 ----
 // TEST[skip:example fragment]
 
-In the above example, we execute each of the two sub searches
-independently of each other. First we run the term query for
-`blue shoes sales` using the standard BM25 scoring algorithm. Then
-we run the text expansion query for `What blue shoes are on sale?`
+In the above example, we execute each of the two `standard` retrievers
```

Comment:
This is another example that could be annotated with numbers in the example fragment?

Reply:
Same comment as above. I didn't find a way where I felt like the readability of the example was improved.

Reply:
Here's along the lines of what I was thinking: 

```
[source,console]
----
GET example-index/_search
{
    "retriever": {
        "rrf": { <3>
            "retrievers": [
                {
                    "standard": { <1>
                        "query": {
                            "term": {
                                "text": "blue shoes sale"
                            }
                        }
                    }
                },
                {
                    "standard": { <2>
                        "query": {
                            "text_expansion":{
                                "ml.tokens":{
                                    "model_id":"my_elser_model",
                                    "model_text":"What blue shoes are on sale?"
                                }
                            }
                        }
                    }
                }
            ],
            "window_size": 50,
            "rank_constant": 20
        }
    }
}
----
// TEST[skip:example fragment]

In the above example, we execute each of the two `standard` retrievers
independently of each other.

<1> First we run the `standard` retriever
specifying a term query for `blue shoes sales` using the standard BM25
scoring algorithm. 
<2> Next we run the `standard` retriever specifying a
text expansion query for `What blue shoes are on sale?`
using our <<semantic-search-elser, ELSER>> scoring algorithm.
<3> The `rrf` retriever allows us to combine the two top documents sets
generated by completely independent scoring algorithms with equal weighting.

Not only does this remove the need to figure out what the appropriate
weighting is using linear combination, but RRF is also shown to give improved
relevance over either query individually.
```

If you don't feel that it provides value though, it's definitely not a necessary change.

Reply:
This is nicer. I will change this. Thank you for the write up here. (It hadn't occurred to me to mark the rrf retriever out of order in this way.)